src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -495,7 +495,6 @@ public String getQuestionResultStatisticsCsv(
         return studentResults;
     }
 
-    @SuppressWarnings("PMD.UnusedPrivateMethod") // false positive by PMD.
```

Comment:
@wkurniawan07 
what should we do with these false positives. In pmd5 we need them for the builds to pass but in pmd6 this bug has been fixed. 

 Should I exclude the rule `UnusedPrivateMethod` in the pmd5 ruleset? (Though it doesn't sound like a good idea).
Or, should we keep these false positives till eclipse has the support for PMD 6?